src/Sylius/Behat/Page/Admin/Currency/IndexPage.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Currency;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPage as BaseIndexPage;
+use Sylius\Behat\Page\ElementNotFoundException;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class IndexPage extends BaseIndexPage implements IndexPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCurrencyDisabled(CurrencyInterface $currency)
+    {
+        return $this->checkCurrencyStatus($currency, 'Disabled');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCurrencyEnabled(CurrencyInterface $currency)
+    {
+        return $this->checkCurrencyStatus($currency, 'Enabled');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function assertAmountOfCurrencies($expectedAmount)
```

Comment:
assert suggests it will throw an exception.

Perhaps we just need a simple method: ``getNumberOfCurrencies``? And change the type of assertion in the context, wdyt?